packages/react-select/src/Select.tsx

```diff
@@ -1293,10 +1293,6 @@ export default class Select<
   onControlMouseDown = (
     event: React.MouseEvent<HTMLDivElement> | React.TouchEvent<HTMLDivElement>
   ) => {
-    // Event captured by dropdown indicator
```

Comment:
is there a test for this particular use case - dropdown indicator preventing default event (https://github.com/JedWatson/react-select/blob/06e34882638d1526b9f5a1238bb567a3e9460ce5/packages/react-select/src/Select.tsx#L1345)

Also, not quite clear why `stopPropagation` is not a case, `onMenuMouseDown` uses it (uses both) for example.

----

This code was added here for a reason, and I can see other pieces relying on this one. Basically - all `.preventDefault` in this file.

What if we try to change the game rules and just rely on a different information source?
```tsx
const createEventTracker = () => {
  const tracker = new WeakSet();
  return {
       mark: (event) => {
        tracker.add(event.target);
        Promise.resolve().then(() => tracker.delete(event.target);
      },
      isMarked: (event) => tracker.has(event.target);
 }
} 

/// ....
const mouseDownTracker = createEventTracker();

onDropdownIndicatorMouseDown = (e) => {
  // ...
  mouseDownTracker.mark(e);
}

onControlMouseDown = (event) => {
    // Event captured by dropdown indicator
    if (mouseDownTracker.isMarked(event)) {
      return;
    }
}
```

Might be a little too much code to support a single line, but this code will keep the old behavior and unlock our usecase.

Reply:
This prevent default was added in https://github.com/JedWatson/react-select/pull/5134 — I can't imagine anything in Select is leaning on it?

We can definitely change the rules like you said if the behaviour is needed. I'm just not sure it is.

Reply:
The failing tests might point to it being needed I suppose ;).

Reply:
Right, and you added tests for #5050 case, and they are passing. May be something else in between, including changes in DatePicker resolved the problem in the meanwhile

Reply:
Yeah you're right, which is what the tests are showing now. I'll dig in soon.